library/std/src/fs.rs

```diff
@@ -2529,7 +2529,7 @@ pub fn copy<P: AsRef<Path>, Q: AsRef<Path>>(from: P, to: Q) -> io::Result<u64> {
 /// limited to just these cases:
 ///
 /// * The `original` path is not a file or doesn't exist.
-///
+/// * The 'link' path already exists.
```

Comment:
Nit: Keep the spacing before the `Examples` header
```suggestion
/// * The 'link' path already exists.
///
```

Reply:
Done in ab274630b9b0accb3b86ca70687240f92d488629